src/js/classes/Table.js

```diff
@@ -245,6 +269,32 @@ export class RankedTable {
   }
 
   render() {
+    // set up search bar
+    const searchMenu = this.container.getElementsByClassName("menu")[0];
+    const searchOptions = this.data.flatMap((row) =>
+      row.flatMap((value, i) =>
+        this.searchCols[i] ? [value.toLocaleString()] : []
+      )
+    );
+    // Current behavior is to alphabetically sort all options,
+    // potentially mixing values from different columns
```

Comment:
Good note, but we're only ever using the search for one column so no need for that.

Reply:
Well, there is the case of the MDJ table where it's not so straightforward - you can search by both County Name and judge name. Whether they are two separate columns or we add the concept of "sub-rows" to Table is yet to be decided, but we may decide to list them separately in the dropdown.

Reply:
Yeah I think this is a design question for Rong as well. I think it'll probably be confusing to have them in the same dropdown, but we can see what she thinks.